Sources/SotoCore/Exports.swift

```diff
@@ -15,6 +15,8 @@
 @_exported import protocol SotoSignerV4.Credential
 @_exported import struct SotoSignerV4.StaticCredential
 
+@_exported import protocol BaggageContext.Context
```

Comment:
Is this export necessary? People only stumble into the `Context` as a parameter to Soco functions right? I don't think they need to use it explicitly?

Reply:
Fair enough. If they aren't creating their then this is unnecessary

Reply:
removed, along with the Logging export